server/raft.go

```diff
@@ -76,6 +76,7 @@ type RaftNode interface {
 	Stop()
 	Delete()
 	Wipe()
+	RecreateInternalSubs() error
```

Comment:
Should we throw away old one and create a new one with same name vs trying to patch on the fly?

Reply:
The advantage of doing it this way is append entries etc that have been received and are being processed will continue to be without interruption, whereas if we have to restart the whole node, we need to flush state to disk for a potentially large number of nodes and pause operations.

Reply:
The new node will do catchup properly as designed.. It might yield a better result, but I am not adamant about this approach vs current one.

Reply:
I did go back and forth on this a little to start with. In the end what I came up with was that you can push an account claim update that changes the account at pretty much any time and the system will honour it.

If you have a busy account with hundreds of assets running, restarting all the Raft groups can create a run on the disk (and dios) by reloading all the peer states, TAVs, WALs etc, whereas actually the only things we really need to update are the sendqs and subs.